### DISCUSSION TITLE
Deskewing and Cropping Scans using ImageMagick - continued

### DISCUSSION THREAD

**Discussion by dprdpr** [ImageMagick/ImageMagick#5886] [Help]
Hello.
I'm trying to use the example command shown at https://legacy.imagemagick.org/discourse-server/viewtopic.php?t=36609
to do what is shown there. I used the example command given and didn't get the results I was hoping for
- right hand side of image has a white strip
- top of image is not aligned.

![snip](https://user-images.githubusercontent.com/18513009/208091131-8f3ed278-d160-4f28-baa4-e30ba987ffc5.JPG)

The version I'm using is ImageMagick 7.1.0-25 Q16-HDRI x64 2022-02-16 https://imagemagick.org on Windows 10.

Any suggestions on what I'm doing wrong?
Thanks

**Comment by fmw42** [comment]
Works fine for me on IM 7.1.0.54 on Mac OSX in Unix syntax

```
magick testscan.png -rotate 90 -set option:color "%[pixel:u.p{10,10}]" -background "%[color]" -deskew 80% +repage \
\( -clone 0 -fuzz 20% -fill black -draw "color 10,10 floodfill" -alpha off +fuzz -fill white +opaque black -type bilevel \
-define connected-components:mean-color=true \
-define connected-components:area-threshold=1000 \
-connected-components 8 -set option:cropbox "%@" +delete \) \
-crop "%[cropbox]" +repage \
edited_scan2.png

```

The Images are too large to post, so I converted them to JPGs and processed again

![testscan](https://user-images.githubusercontent.com/13458218/208147233-8483b610-eca2-4ee6-9193-24832aa819b1.jpg)


```
magick testscan.jpg -rotate 90 -set option:color "%[pixel:u.p{10,10}]" -background "%[color]" -deskew 80% +repage \
\( -clone 0 -fuzz 20% -fill black -draw "color 10,10 floodfill" -alpha off +fuzz -fill white +opaque black -type bilevel \
-define connected-components:mean-color=true \
-define connected-components:area-threshold=1000 \
-connected-components 8 -set option:cropbox "%@" +delete \) \
-crop "%[cropbox]" +repage \
-type truecolor edited_scan2.jpg

```

![edited_scan2](https://user-images.githubusercontent.com/13458218/208147278-fe1237be-4da5-4e62-9d10-42326bb8a668.jpg)


The syntax on Windows is different. Remove the \ in front of the opening and closing parentheses and change the line ending \ to ^. If in .bat script then double the %

**Reply by dprdpr** [reply]
Thanks for the quick reply, 

I upgraded to the latest version (Win64 static at 16 bits-per-pixel component with [high dynamic-range imaging](https://imagemagick.org/script/high-dynamic-range.php) enabled)

I downloaded the input jpg and and ran it through the command you posted. My result has the right strip. Both your output jpg and mine both have the skew along the top. Attached is my output jpg.
![edited](https://user-images.githubusercontent.com/18513009/208157052-e4022faa-1403-49f1-b9f7-b6ac8d4515dc.jpg)

**Reply by fmw42** [reply]
If the top edge is not perfectly straight, it will look like it is skewed. You can try adjusting the arguments. Or you can trim to make it straight.

**Comment by dprdpr** [comment]
ok. I was just trying to prove the example before scanning my own
